logdna/resource_view.go

```diff
@@ -299,6 +333,7 @@ func resourceView() *schema.Resource {
 						"immediate": {
 							Type:     schema.TypeString,
 							Optional: true,
+							Default:  "false",
```

Comment:
do we know why these are strings and not `schema.TypeBool`? I believe this came up before but can't remember. not sure we could change it without a breaking change for state anyway.

Reply:
It feels like a mistake.  The confusion might have happened when trying to figure out these types based on our APIs.  It my testing I've discovered that the *requests* (PUTs) can accept string OR booleans!  The responses all seem to have booleans, though, so we're just inconsistent everywhere.

This default was needed to fix the fact that users can configure TF with strings, but the `resourceRead` will returns bools.

Reply:
bleh, ok 👍